catalogue/webapp/services/labs/image-similarity.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import fetch from 'isomorphic-unfetch';
+
+type SimilarityMetric = 'palette' | 'feature';
+
+const ROOT_URL = 'https://labs.wellcomecollection.org';
+const SIMILARITY_METRIC: { [key: SimilarityMetric]: string } = {
```

Comment:
I know metric makes sense but, and maybe it's because I'm working on reporting, it feels overloaded.
Could we use service / something else?